tests/selenium/test_basic_copy_paste.py

```diff
@@ -85,6 +85,7 @@ def test_tbcp001_copy_paste_callback(test):
 
     assert target.cell(1, 0).get_text() == "0"
     assert target.cell(1, 1).get_text() == "MODIFIED"
+    assert len(test.get_log_errors()) == 0
```

Comment:
```suggestion
    assert test.get_log_errors() == []
```
That way if it fails you see what the error was

Reply:
Good point https://github.com/plotly/dash-table/pull/841/commits/9e8028981e433d91ecd3fe4aa7bea16b3a7934f7